packages/frontend/src/components/Info/Accordion.tsx

```diff
@@ -24,33 +35,35 @@ export const InfoAccordion = () => {
           </StyledHeader>
           <StyledContent>
             <RuleText>
-              Total number of 100 raffle tickets is devided between auction pool and raffle pool. Auction winners will
-              win a Voucher Code that is redeemable for a Free Devcon ticket. Raffle winners will receive a voucher code
-              that is redeemable for 1 Devcon Ticket at Full Price.
+              Total number of {totalCount} raffle tickets is devided between auction pool and raffle pool. Auction
+              winners will win a Voucher Code that is redeemable for a Free Devcon ticket. Raffle winners will receive a
+              voucher code that is redeemable for 1 Devcon Ticket at Full Price.
             </RuleText>
             <Rule
-              heading="Auction pool: 20"
+              heading={`Auction pool: ${auctionWinnersCount}`}
               rule="Tickets from auction pool will be distributed to the highest bidding participants. The cost of a Devcon
               ticket in that pool is equal to the bid. All proceeds will go towards X Public Good."
-              example="You bid 0.5 ETH and end up in top 20 of bidders. You receive the ticket for 0.5 ETH."
+              example={`You bid ${exampleBid * 2.5} ETH and end up in top ${auctionWinnersCount} of bidders. You receive the ticket for 
```

Comment:
Add this calculation for using only one example variable and making it always bigger for the auction example.